automation/pr-to-meta-resin.sh

```diff
@@ -48,12 +48,13 @@ docker-resin-supervisor-disk: Update to ${TAG}
 
 Changelog-Entry: Update supervisor to ${TAG}
 Change-Type: patch
-" <<EOF
+"
+
+	git push origin $HEAD <<EOF
```

Comment:
Nice one! I changed it to use it in the push cause when cloning then the pasword becomes visible when doing `git remote -v`.

I also realized that hub picks up `$GITHUB_USER` and `$GITHUB_PASSWORD` so I changed it to use that instead.